OrbitQt/orbitmainwindow.cpp

```diff
@@ -937,6 +939,22 @@ void OrbitMainWindow::on_actionToggle_Capture_triggered() { app_->ToggleCapture(
 
 void OrbitMainWindow::on_actionClear_Capture_triggered() { app_->ClearCapture(); }
 
+void OrbitMainWindow::on_actionCaptureOptions_triggered() {
+  static const QString kCollectThreadStates{"CollectThreadStates"};
+  QSettings settings;
+
+  auto dialog = std::make_unique<orbit_qt::CaptureOptionsDialog>(this);
```

Comment:
Since `dialog` does not leave this local scope, this should be stack-allocated.

Reply:
Ah yes, done.